docs/BigQueryTable-batchsink.md

```diff
@@ -53,6 +53,11 @@ is ignored if the table already exists.
 * When this is set to true, table will be created with required partition filter. 
 * When this is set to false, table will be created without required partition filter.
 
+**Clustering order**: Comma separated list of fields that determines the sort order of the data. 
```

Comment:
Clustering Order

Reply:
Should use the csv widget so that users don't need to worry about a delimiter.

Reply:
Should document any restrictions, such as the supported data types and if there is a max number of fields. Are they allowed to be nullable?

Reply:
Also, should make it clear that this is only used when the BigQuery table is automatically created. It is ignored if the table already exists.